src/klipse/compiler.cljs

```diff
@@ -102,9 +102,12 @@
 (defn create-editor [compiler config]
   (let [editor (js/CodeMirror.fromTextArea
                   (js/document.getElementById "code") 
-                  (clj->js config))] 
+                  (clj->js config))
+        idle-time 3000
+        fn-process #(process-input compiler (get-value editor))] 
+    (.on editor "change" (debounce fn-process idle-time false))
```

Comment:
when using `cljs_in` url parameter it doesn't eval until the user edits the input.
It should eval without requiring user interaction.